browser/actions/unclaimPrrr.js

```diff
@@ -1,8 +1,5 @@
-import request from '../request'
-import state from '../state'
-import loadPrrrs from './loadPrrrs'
+import socket from '../socket'
 
 export default function unclaimPrrr(prrrId) {
-  return request('post', `/api/pull-request-review-requests/${prrrId}/unclaim`)
-    .then(loadPrrrs)
+  socket.emit('unclaimPrrr', prrrId)
```

Comment:
I'd prefer all socket message payloads to be an object. `socket.emit('unclaimPrrr', {prrrId})`